tasks/configure_option_merge.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Read {{ item.key }}"
+  become: yes
+  become_user: www-data
+  shell: "wp option get {{ item.key }} --path=/var/www/wordpress --format=json"
+  changed_when: False
+  register: result
+
+- name: Read output form json to yaml
+  set_fact:
+    option_plugin: "{{ result.stdout | from_json }}"
+
+- name: Merge settings
```

Comment:
You can combine the three fact tasks.

Reply:
no i can not since im using the var that i set before and ansible set_fact does not say anything about execution order

Reply:
But the task have an execution order?
What I mean is that you just set the fact to use it once, exactly after it. I think you could use one task instead of all three:
```yml
- name: Write merged settings to temporary file
   copy:
     content: "{{ result.stdout | from_json | combine( item.value ) | to_json }}"
     dest: "{{ tmp_file.path }}"
   changed_when: False
```